### PR TITLE
Add all reviewer teams

### PR DESCRIPTION
**Pull request by lambdanis**
So far becoming a reviewer was impossible - [the doc](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md#reviewer) says that to become a reviewer you have to
```
Open a PR in the member management file for at least two teams: "Reviewers" and for a specific review area
```
but the teams weren't listed here. This PR fixes that, so that new reviewers can be added.

### CODE REVIEW

File: ladder/teams/ebpfio.yaml
Comment (xmulligan) [inline ladder/teams/ebpfio.yaml:1]: I don't think this is needed anymore since ebpf.io isn't under Cilium

Comment (joestringer) [reply ladder/teams/ebpfio.yaml:1]: Yeah true, we could drop this specific file.

File: ladder/teams/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+members:
```
Comment (joestringer) [inline ladder/teams/ci.yaml:1]: Where did this team come from?

File: ladder/teams/ebpf-bsc.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+members:
+- borkmann
+- brendangregg
+- dthaler
+- sinkap
```
Comment (joestringer) [inline ladder/teams/ebpf-bsc.yaml:5]: We don't need this one any more.

File: ladder/teams/ci.yaml (hunk 1 above)
Comment (lambdanis) [reply ladder/teams/ci.yaml:1]: typo, it's a duplicate of `cli`. Will remove.

**Review by joestringer — APPROVED** [review]
Thanks!

**Pull request merged**